crates/onboarding/src/ai_setup_page.rs

```diff
@@ -188,6 +188,11 @@ fn render_llm_provider_card(
                                 workspace
                                     .update(cx, |workspace, cx| {
                                         workspace.toggle_modal(window, cx, |window, cx| {
+                                            telemetry::event!(
+                                                "Welcome AI modal open",
```

Comment:
Can we update this to match our telemetry format?

Should be "<Noun> <Verb>ed", written in Title Case.

```suggestion
                                                "Welcome AI Modal Opened",
```

Reply:
Yes! I didn't know we had a telemetry format so thank you for letting me know. I'll keep that in mind for any more I add in the future